lib/utils.ts

```diff
@@ -61,6 +61,7 @@ export const trackImage = (
           name,
           imageBase64,
           browser: Cypress.browser.name,
+          viewport: options?.viewport ? options.viewport : Cypress.config("viewportWidth") * pixelRatio + "x" + Cypress.config("viewportHeight") * pixelRatio,
```

Comment:
you could use `??` [operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)
```suggestion
          viewport: options?.viewport ?? `${Cypress.config("viewportWidth") * pixelRatio}x${Cypress.config("viewportHeight") * pixelRatio}`,
```